include/clap/plugin-id.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+// Pair of plugin ABI and plugin identifier.
+typedef struct clap_plugin_id {
+   // The plugin ABI name, in lowercase.
+   // eg: "clap"
```

Comment:
I think this needs more expansion on what you mean by "ABI Name"; or alternately a set of constexpr char* with examples.

Reply:
Do you think it is OK to give VST3 and VST2 examples in the documentation?